keyboards/adkb96/rev1/rules.mk

```diff
@@ -0,0 +1,2 @@
+BACKLIGHT_ENABLE = no
```

Comment:
You don't need this file at all. Both of these rules will be inherited from

https://github.com/qmk/qmk_firmware/blob/58dbd49fa4a2be143b49ee1c8691789f45701e10/keyboards/adkb96/rules.mk#L63-L64

You only need this file if you need to override rules that are set further up the directory tree.